bft/testers/cudf/runner.py

```diff
@@ -23,7 +41,14 @@ def type_to_cudf_dtype(type: str):
 
 
 def literal_to_str(lit: CaseLiteral):
```

Comment:
Consider turning this into an __str__ function on CaseLiteral so that you can just str(lit) or use print functions.